.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.19
+
+      - name: Build
+        run: |
+          cd ./cmd/mnemonikey
+          go build -v ./...
```

Comment:
There's no sense running `go build` unless we cache or upload the resulting binary somewhere. Probably it is better to just run `go vet` and `go test` (or `go test -vet=all` to do both at once)

Reply:
I removed the `build` action.